training/src/anemoi/training/config/model/gnn.yaml

```diff
@@ -2,7 +2,7 @@ activation: GELU
 num_channels: 512
 
 model:
-  _target_: anemoi.models.models.encoder_processor_decoder.AnemoiModelEncProcDec
+  _target_: anemoi.models.models.AnemoiModelEncProcDec
```

Comment:
Why is this changing here?

Reply:
Now the different model classes are included in the `__all__` attribute and I think it may be more friendly for the user to access each model class directly from `anemoi.models.models`. But it may have some negative implications I am not aware of, so feel free to raise any concerns